ScaleformUI_Lua/src/scaleforms/PauseMenu/PauseMenuScaleform.lua

```diff
@@ -113,19 +113,19 @@ function Pause:AddRightListLabel(tab, leftItem, label)
 end
 
 function Pause:AddRightStatItemLabel(tab, leftItem, label, rightLabel)
-    self._pause:CallFunction("ADD_RIGHT_LIST_ITEM", false, tab, leftItem, 1, 0, label, enabled, rightLabel)
+    self._pause:CallFunction("ADD_RIGHT_LIST_ITEM", false, tab, leftItem, 1, 0, label, nil, rightLabel)
```

Comment:
you can't send null / nil values to scaleforms.. send true in this case

Reply:
it was a nil value,I just change it to nil
enabled = nil
nil = nil
I will test it and put right value

Reply:
instead of nil send a true
`self._pause:CallFunction("ADD_RIGHT_LIST_ITEM", false, tab, leftItem, 1, 0, label, true, rightLabel)`

Reply:
I did it